### DISCUSSION TITLE
/etc/init.d/antmedia: 31: ulimit: error setting limit (Operation not permitted)

### DISCUSSION THREAD

**Discussion by jmax07** [ant-media/Ant-Media-Server#6002] [Q&A]
How to solve this? (SYSTEM: Android 13 with Termux app / Ubuntu 22.04)
###
JAVA_HOME : /usr/lib/jvm/java-11-openjd-amd64/
PLATFORM:linux-aarch64                                  /etc/init.d/antmedia: 31: ulimit: error setting limit (Operation not permitted)                                 
Stopping the Ant Media Service...
Running on  Linux                                       
Stopping Ant Media Server
Selected libraries: (218 items)                         
The Ant Media Service has stopped.
/etc/init.d/antmedia: 31: ulimit: error setting limit (Operation not permitted)
Starting the Ant Media Service...                       
The Ant Media Service has started.
Ant Media Server is installed and started.

~# sudo service antmedia status
/etc/init.d/antmedia: 31: ulimit: error setting limit (Operation not permitted)

**Comment by muratugureminoglu** [comment]
Hi @jmax07 

We recommend Ant Media Server to run on 4 x CPU-Optimized systems and unfortunately, installation with Termux on Android will not be optimal in terms of performance. 

And as to your question, Proot-distro does not provide real root privileges, so the kernel won't give you capabilities to elevate limits. That's exactly why you're getting such an error. Maybe you can try it by extracting the Ant Media Server zip file (after running the installation script, because it will install some required packages) and running the `./start.sh` file.

Regards.

**Comment by jmax07** [comment]
This means it could be possible.
Can you explain how this procedure of extracting the zip and running ./start.sh works?

**Reply by jmax07** [reply]
I ACHIEVED!!! It worked... or almost...
AMS started, but when you press play on "star broadcast", it starts normally, but on YouTube the live does not start.
I tested using AMS through WSL2 on Windows10 and the same problem happened! What is happening?!

I don't have SSL configured with a domain (I don't have a domain to use). Can be it?

**Reply by jmax07** [reply]
Living and learning!
RTMP broadcasts on YouTube must have audio to work. It seems OBVIOUS, but I was testing the transmission with a piece of video without audio. And the live never started!
Here's a lesson for everyone! AMEN!

**Reply by yashtandon113** [reply]
@jmax07
Thanks for updating the case and sharing the information with everyone.

Yes, it is expected behavior that their should be audio in a stream. We have updated this information in the document. You can check it [here](https://deploy-preview-196--ant-media-documentation.netlify.app/guides/publish-live-stream/simulcasting/#how-to-publish-live-stream-on-youtube).

**Comment by jmax07** [comment]
Case closed!
